src/app/invitations/api/index.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const express = require('express');
+const apiAuth = require('login.dfe.api.auth');
+const config = require('./../../../infrastructure/config')();
+const putUpsertInvitation = require('./putUpsertInvitation');
+
+const assert = require('assert');
+
+const router = express.Router();
+
+const routeExport = () => {
+  // Add auth middleware.
+  router.use(apiAuth(router, config));
+
+  assert(config.invitations.redisUrl, 'the invitations.redisUrl config property must be set');
+
+  // Map routed to functions.
+  router.put('/upsert', putUpsertInvitation);
```

Comment:
I think this route should either be be /:invite_id. feels 'more restful'. alternatively a post to /, but post does not imply the upsertyness